index.md

```diff
@@ -1,579 +1,403 @@
-<!--
-  Copyright 2013-2017 Ray Tsang
-  
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-  
-    http://www.apache.org/licenses/LICENSE-2.0
-  
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
--->
-
-JDeferred
-=========
-
-JDeferred is a Java Deferred/Promise library similar to JQuery's Deferred Object.
-
-Inspired by [JQuery](https://github.com/jquery/jquery) and [Android Deferred Object](https://github.com/CodeAndMagic/android-deferred-object).
-
-<a name="features"></a>Features
---------
-* Deferred object and Promise
-* Promise callbacks
-  * ```.then(…)```
-  * ```.done(…)```
-  * ```.fail(…)```
-  * ```.progress(…)```
-  * ```.always(…)```
-* Multiple promises
-  * ```.when(p1, p2, p3, …).then(…)```
-  * ```.race(p1, p2, p3, …).then(…)```
-  * ```.settle(p1, p2, p3, …).then(…)```
-* Callable and Runnable wrappers
-  * ```.when(new Runnable() {…})```
-  * ```.race(new Runnable() {…})```
-  * ```.settle(new Runnable() {…})```
-* Uses Executor Service
-* Java Generics support
-  * ```Deferred<Integer, Exception, Double> deferred;```
-  * ```deferred.resolve(10);```
-  * ```deferred.reject(new Exception());```
-  * ```deferred.notify(0.80);```
-* Android Support
-* Java 8 Lambda friendly
-* Yes it's on Maven Central Repository!
-
-Maven
------
-```xml
-<dependency>
-   <groupId>org.jdeferred2</groupId>
-   <artifactId>jdeferred-core</artifactId>
-   <version>${version}</version>
-</dependency>
-```
-
-Gradle
------
-```
-compile 'org.jdeferred2:jdeferred-android-aar:${version}'
-// or
-compile 'org.jdeferred2:jdeferred-android-aar:${version}@aar'
-```
-
-Find available versions on [Maven Central Repository](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.jdeferred%22%20AND%20a%3A%22jdeferred-core%22).
-
-Please note that JDeferred 2.x is published under the `org.jdeferred2` group and uses `org.jdeferred2` as base package name.
-Use the following coordinates if you're looking for an older version of JDeferred
-
-Maven
------
-```xml
-<dependency>
-   <groupId>org.jdeferred</groupId>
-   <artifactId>jdeferred-core</artifactId>
-   <version>${version}</version>
-</dependency>
-```
-
-Gradle
------
-```
-compile 'org.jdeferred:jdeferred-android-aar:${version}'
-// or
-compile 'org.jdeferred:jdeferred-android-aar:${version}@aar'
-```
-
-<a name="compatibility"></a>Compatibility

```

Comment:
this section is missing in the new version :'(

Reply:
ah yeah, i made a copy to put in 2.x branch. just realized it's in my fork :)

Reply:
in the mean time, let's try to keep the README.md as the source of doc, since we don't have easier way to do this right now :(